apps/ui/components/reservation-unit/QuickReservation.tsx

```diff
@@ -286,17 +278,14 @@ type AvailableTimesProps = {
 function getAvailableTimesForDay({
   day,
   duration,
-  time,
   isSlotReservable,
   reservationUnit,
   fromStartOfDay = false,
 }: AvailableTimesProps): string[] {
   if (reservationUnit == null) return [];
   const [durationHours, durationMinutes] =
     duration?.split(":").map(Number) || [];
```

Comment:
This is bit iffy now that `label !== value` pretty sure it's gonna cause issues later.

Reply:
Hmm... Not sure I agree here. What was asked for was that the durations would be _displayed_ in the same manner everywhere. Changing only the `label` leaves everything else in place logic-wise === everything works. Or "works", but as well as it used to before the change.

As I see it the only change is that the user sees the time in the requested format. Changing the value to minutes (as suggested below) is a good idea, but not necessary for these changes. Or that's my two cents at least.